server/controllers/api/videos/import.ts

```diff
@@ -219,9 +221,8 @@ async function addYoutubeDLImport (req: express.Request, res: express.Response)
     videoImportId: videoImport.id,
     generateThumbnail: !thumbnailModel,
     generatePreview: !previewModel,
-    fileExt: youtubeDLInfo.fileExt
-      ? `.${youtubeDLInfo.fileExt}`
-      : '.mp4'
+    fileExt: `.${youtubeDLInfo.ext || 'mp4'}`,
+    mergeExt: youtubeDLInfo.mergeExt && `.${youtubeDLInfo.mergeExt}`
```

Comment:
I prefer to explicitly set the default option. If `youtubeDLInfo.mergeExt === ''` then we send `''` as option. A ternary would allow us to choose this default value (`undefined` for example).

Reply:
Indeed this is a much better practice!